spec/system/candidate_interface/submitting/candidate_tries_to_submit_to_undergraduate_course_without_a_levels_spec.rb

```diff
@@ -0,0 +1,98 @@
+require 'rails_helper'
```

Comment:
Rename file to remove `to_`

`candidate_tries_to_submit_undergraduate_course_without_a_levels_spec.rb`

Reply:
Done